lib/rules/color-named/index.js

```diff
@@ -70,6 +71,10 @@ const rule = function (expectation, options) {
           return false
         }
 
+        if (!isStandardSyntaxFunction(node)) {
+          return false
```

Comment:
This doesn't appear to work. I don't know a _ton_ about `postcss-value-parser`, but I believe this is because returning  `undefined` on a `value` would work. However, since we are making an assertion on a `node`, you need to return false for it to skip the `value`s within that `node`. Otherwise, I would group it with the `isStandardSyntaxValue` check below this.